Backend/app/dashboard/CommonAPIs/src/Dashboard/Common.hs

```diff
@@ -52,6 +52,10 @@ data FarePolicy
 
 data DriverHomeLocation
 
+data DriverGoHomeRequestStatus
```

Comment:
where are you using this data type ??

Reply:
Oh I was initially using this but then I removed this ones use. I will remove this as well.